packages/@react-spectrum/s2/style/spectrum-theme.ts

```diff
@@ -603,7 +618,11 @@ export const style = createTheme({
     borderEndWidth: createRenamedProperty('borderInlineEndWidth', borderWidth),
     borderTopWidth: borderWidth,
     borderBottomWidth: borderWidth,
-    borderStyle: ['solid', 'dashed', 'dotted', 'double', 'hidden', 'none'] as const,
+    borderStartStyle: createRenamedProperty('borderInlineStartStyle', borderStyle),
+    borderEndStyle: createRenamedProperty('borderInlineEndStyle', borderStyle),
+    borderTopStyle: borderStyle,
+    borderBottomStyle: borderStyle,
+    borderStyle: borderStyle,
```

Comment:
This will break when both the shorthand and longhand properties are set. We would need to expand `borderStyle` to all 4 properties in all cases. But I don't think we need these individual properties since we already have individual width properties. I don't think we'll need borders with different styles on each side.